### DISCUSSION TITLE
Pause and resume a timer.

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1766] [Help] [answered]
Hey, I want to make a static timer which I can pause and resume. How can I do this?
I know how to make the timer static, but I want to know how I can pause/resume.

edit: Im talking about this timer: 
```
MinecraftServer.getSchedulerManager().submitTask {
            TaskSchedule.tick(ticks)
        }
```

**Comment by ZakShearman** [comment, marked as answer]
Are you talking about a TaskSchedule or an arbitrary timer you're implementing?

**Reply by DevBlokJochem** [reply]
Im talking about the TaskSchedule. Sorry.

**Reply by ZakShearman** [reply]
Late reply but you're going to have to store the task schedule, cancel it and then track the time remaining based on the time that has already passed.

TaskSchedules themselves don't have a pausing functionality, so you have to replicate it yourself.
